src/main/scala/Utils.scala

```diff
@@ -0,0 +1,19 @@
+
+import com.typesafe.scalalogging.LazyLogging
+import com.codahale.metrics.{Metric, MetricFilter}
+import scala.util.matching.Regex
+
+object Utils extends LazyLogging {
```

Comment:
Instead of calling this `utils` perhaps it could be called 'CloudWatchMetricFilter`